File: ShadhinMusicLibrary/src/main/java/com/shadhinmusiclibrary/data/model/lastfm/Tags.java
package com.shadhinmusiclibrary.data.model.lastfm;

import androidx.annotation.Keep;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

@Keep
public class Tags {
    @SerializedName("tag")
    @Expose
    private List<Tag> tag = null;

    public List<Tag> getTag() {
        return tag;
    }

    public void setTag(List<Tag> tag) {
        this.tag = tag;
    }
}

File: ShadhinMusicLibrary/src/main/java/com/shadhinmusiclibrary/data/model/lastfm/Tags.kt
package com.shadhinmusiclibrary.data.model.lastfm

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName
import com.google.gson.annotations.Expose

@Keep
internal class Tags {
    @SerializedName("tag")
    @Expose
    var tag: List<Tag>? = null
}